src/App.js

```diff
@@ -144,7 +144,6 @@ const POKEMONS = [
 ]
 
 function App() {
-  console.log({css})
```

Comment:
Commit name is `change App.module.css` but there is no changes in [this commit](https://github.com/ohlabiak/react_2022/pull/3/commits/d2b9a28221d2678fdbb266481bec8ed5d13d18bc).
when you see git status before commit you can see all changed files.

The only one deleting of `console log` can be done inside another commit. 
Better is do not mention SO IMPORTANT ACTION :-)